src/platform_impl/linux/window.rs

```diff
@@ -594,7 +603,7 @@ impl Window {
   }
 
   pub fn is_maximizable(&self) -> bool {
-    true
+    self.window.is_resizable() && self.maximized.load(Ordering::Acquire)
```

Comment:
`is_maximizable` is not the same as `is_maximized`

Reply:
Oh, I thought that would affect the `maximized` state. I will revert it back.

Reply:
changed it back to `true`.